app.py

```diff
@@ -0,0 +1,148 @@
+import json
+from flask import Flask, redirect, render_template, flash, session, request
+from models import User, connect_db, db
+import os
+from secret import SECRET_KEY, OPEN_CHARGE_MAP_KEY, MAP_KEY
+from forms import SignUpForm, LoginForm
+from sqlalchemy.exc import IntegrityError
+import requests
+
+app = Flask(__name__)
+
+app.config["SECRET_KEY"] = os.environ.get('SECERT_KEY', SECRET_KEY)
+app.config["SQLALCHEMY_DATABASE_URI"] = os.environ.get(
+    'DATABASE_URL', "postgresql://localhost/chargR?user=postgres&password=postgresql")
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+app.config['SQLALCHEMY_ECHO'] = True
+app.config['DEBUG_TB_INTERCEPT_REDIRECTS'] = False
+
+
+API_BASE_URL = 'https://api.openchargemap.io/v3/poi/'
+MAP_BASE_URL = 'http://open.mapquestapi.com/geocoding/v1/'
+
+# toolbar = DebugToolbarExtension(app)
+
+connect_db(app)
+db.create_all()
+
+
+@app.route('/')
+def start():
+    return redirect('/home')
+
+
+@app.route('/home')
+def home_page():
+    return render_template('home.html')
+
+########################### Location And Coords ######################################
+
+
+def get_coords(location):
+    """Get the location and return Latitude and Longitude """
+
+    res = requests.get(f'{MAP_BASE_URL}/address',
+                       params={'key': MAP_KEY, 'location': location})
+    data = res.json()
+    lat = data['results'][0]['locations'][0]['latLng']['lat']
+    lng = data['results'][0]['locations'][0]['latLng']['lng']
+    coords = {'lat': lat, 'lng': lng}
+    return coords
+
+
+def get_results(data):
+    """turn json into python"""
+
+    res = json.loads(data.text)
+    return res
+
+
+def get_info(coords):
+    """Get the coords and return data """
+    latitude = coords['lat']
+    longitude = coords['lng']
+
+    response = requests.get(f'{API_BASE_URL}', params={'key': OPEN_CHARGE_MAP_KEY,
+                            'countrycode': 'US', 'latitude': latitude, 'longitude': longitude})
```

Comment:
Don't forget to specify somewhere (in the README?) that this app will only work for USA.